src/Command/MakerCommand.php

```diff
@@ -88,7 +90,17 @@ protected function interact(InputInterface $input, OutputInterface $output)
 
     protected function execute(InputInterface $input, OutputInterface $output)
     {
-        $generator = new Generator($this->fileManager, 'App\\');
+        if (!$this->fileManager->isNamespaceConfiguredToAutoload($this->rootNamespace)) {
+            throw new \LogicException(
+                sprintf(
+                    'It looks like your app may be using a namespace other than "%s".'.
+                    ' See https://symfony.com/doc/current/bundles/SymfonyMakerBundle/index.html for how to configure this.',
+                    $this->rootNamespace
+                )
+            );
+        }
```

Comment:
Instead of an exception, let's convert this to a warning. Something like:

```php
$this->io->note([
    sprintf('It looks like your app may be using a namespace other than "%s".', $this->rootNamespace),
    'See https://symfony.com/doc/current/bundles/SymfonyMakerBundle/index.html#configuration to configure this.
]);
```

The `#configuration` part of the link doesn't exist yet, but we'll add it.